.github/workflows/ci.yml

```diff
@@ -9,19 +9,19 @@ jobs:
         ocaml-compiler:
           - 4.13.x
           - 4.14.x
+          - 5.0.x
```

Comment:
Making kappa binaries compile with OCaml 5 is a mandatory step that has to be done and is well done here but I fear that sadly, the journey to make KaSim "compatible" with OCaml5(while keeping it compatible with OCaml 4) is more complex than that! 
I'll confirm my fears tomorrow (it's as simple as running `make check`) but OCaml 5 changes the random number generator provided by the standard library (to make it multithread capable) so `--seed 42` does not gives the same simulation when compiled with Ocaml 4 and OCaml 5 :-(

Reply:
I can confirm essentially the entire test suite fails when `make check`'ed under OCaml 5

Reply:
Testing the test suite on OCaml 4 and OCaml 5 with the same reference files does not make sense. 
The random generators are not the same, so most stochastic parts will differ.